src/components/Label/Label.module.scss

```diff
@@ -29,77 +33,115 @@
   transform-origin: bottom left;
 }
 
-.kindFill.colorPrimary {
-  background-color: var(--primary-color);
-}
+.kindFill {
+  &.colorPrimary {
+    background-color: var(--primary-color);
 
-.kindFill.colorPrimary svg path {
-  fill: var(--primary-color);
-}
+    svg path {
+      fill: var(--primary-color);
+    }
 
-.kindFill.colorDark {
-  background-color: var(--inverted-color-background);
-}
+    &.clickable:hover,
+    &.clickable:active {
+      background-color: var(--primary-hover-color);
```

Comment:
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/acc9503d-d4ac-4e78-98a6-f9dda454901f)
Hovered outlined version doesn't look too good in dark theme. Though token seems correct... I'd consult with the designers just to be sure it's okay - maybe we need to change text color as well e.g.

Reply:
Talked to Yael, nothing to do at the moment, this is by design although contrast is not that great, I will raise an issue in the team meeting about color contrast